src/App.tsx

```diff
@@ -408,6 +409,14 @@ export class App extends React.PureComponent<unknown, AppState> {
 		this.setPage(nextPage);
 	}
 
+	handleChangeScopeTabs(newValue: number){
```

Comment:
Can remove this method. Pass this.setPage into <'ScopeTabs'> instead. Change return type to symbol. Remove  ScopeTabsProps type (no difference between that and the other interface?). Change newValue / value naming in places where the context doesn't tell you exactly what it is.